### DISCUSSION TITLE
Made an app - now how do I make it publicly viewable?

### DISCUSSION THREAD

**Discussion by ProfessorManhattan** [AntlerVC/firetable#1349] [Support / Q&A] [answered]
Hey, I made an app that scans every GitHub Awesome list and compiles them together into a Rowy-sortable-filterable table. However, I can't figure out how to make the list publicly viewable. I enabled Anonymous access but when the user signs in nothing is shown. Also, if they decide to verify with another method they still can't see the table.

I filed a similar issue like a year ago and was told to modify the Firebase permissions so I did that and even if I make everything publicly writable, a user still has to login to access the table.

Is it possible to make a feature that publishes the table to a publicly viewable URL? Perhaps a way of embedding the chart would be nice too so I can use Rowy for Docusaurus charts.

I really wanna publish this all-encompassing GitHub Awesome list because it'd be awesome. Thanks for all your hard work on Rowy --- its cool.

**Comment by harinij** [comment]
Hi @ProfessorManhattan publically embeddable or viewable table is a feature on our roadmap but is currently not available. We will look into expediting that.

**Comment by ProfessorManhattan** [comment]
Even if people could login anonymously it would be usable... that might be
a bit easier to implement.

On Thu, Mar 30, 2023 at 11:58 PM Harini Janakiraman <
***@***.***> wrote:

> Hi @ProfessorManhattan <https://github.com/ProfessorManhattan> publically
> embeddable or viewable table is a feature on our roadmap but is currently
> not available. We will look into expediting that.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rowyio/rowy/issues/1169#issuecomment-1491252162>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOJRHXM2EOUTY56Z5JPUUNDW6ZI7PANCNFSM6AAAAAAWDQBO64>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Regards,

Brian Zalewski
(412) 345-3338

**Comment by ProfessorManhattan** [comment]
Hey @harinij -- how can I monitor the progress of this? Also, I don't mind helping out developing this feature. We might be able to recycle the view and hide all the admin-section pieces of the UI.

**Comment by bofeiw** [comment, marked as answer]
Hi @ProfessorManhattan, you can monitor our upcoming features in our public roadmap: https://roadmap.rowy.io/roadmap. For this specific feature, it is still in our backlog as we have other focuses right now. However we are open for discussion, suggestion, contributions and PRs.
